src/shared/components/paginationControls/PaginationControls.tsx

```diff
@@ -139,7 +140,8 @@ export class PaginationControls extends React.Component<IPaginationControlsProps
         } else {
             return (<span
                         key="textBetweenButtons"
-                        className={"btn btn-default textBetweenButtons disabled" + styles["default-cursor"]}
+                        className={"btn btn-default textBetweenButtons disabled " + styles["default-cursor"]}
```

Comment:
we should be using classnames module for this kind of thing (classnames(class1, class2, class3)